Cannot build on Mac

I followed the readme instructions carefully.

1. To install MacPorts: https://guide.macports.org/#installing
2. Check port version:
  `port version`
  `Version: 2.5.4`
3. Execute the command:
  `port install autoconf automake libtool pkgconfig zlib wxWidgets-3.0`
  retry command, return:
--->  Computing dependencies for autoconf
--->  Cleaning autoconf
--->  Cleaning automake
--->  Cleaning libtool
--->  Computing dependencies for pkgconfig
--->  Cleaning pkgconfig
--->  Cleaning zlib
--->  Computing dependencies for wxWidgets-3.0
--->  Cleaning wxWidgets-3.0
--->  Scanning binaries for linking errors
--->  No broken files found.
--->  No broken ports found.

4. Build MediaInfo CLI:
`cd $BUILD_DIR`
`git clone https://github.com/MediaArea/MediaInfo.git`
`cd MediaInfo/Project/GNU/CLI`
`./autogen.sh`

5. Execute the command:
  `./configure --enable-staticlibs`
  but return last lines:
  `checking dynamic linker characteristics... darwin18.2.0 dyld`
  `checking how to hardcode library paths into programs... immediate`
  `./configure: line 15806: libzen-config: command not found`
  `configure: error: libzen configuration is not found`

What should I do? 
Help

--- a/README.md
+++ b/README.md
@@ -25,6 +25,11 @@ https://guide.macports.org/#installing
 port install autoconf automake libtool pkgconfig zlib wxWidgets-3.0
 ```
 
+*MediaArea tools*
+* [libzen](#zenlib)
+* [libmediainfo](#mediainfolib)
+
+
 ### Dependencies under Linux
 
 #### Listing

